packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -212,13 +212,26 @@ class Serverpod {
   /// started.
   late final ClusterManager cluster;
 
+  /// Define CORS headers to allow cross-origin requests.
+  final Map<String, dynamic> headers;
+
   /// Creates a new Serverpod.
   Serverpod(
     List<String> args,
     this.serializationManager,
     this.endpoints, {
     this.authenticationHandler,
     this.healthCheckHandler,
+    this.headers = const {
+      'Access-Control-Allow-Origin':
+          '*', // Allow all origins or specify a specific origin.
+      'Access-Control-Allow-Methods':
+          'GET, POST, PUT, DELETE', // Add allowed methods.
+      'Access-Control-Allow-Headers':
+          'Content-Type', // Add allowed headers if needed.
+      'Access-Control-Max-Age':
```

Comment:
Doesn't this risk requests being cached by browsers? Which would be bad as only static content should be cached.